camel/utils/token_counting.py

```diff
@@ -21,7 +21,9 @@
 from typing import TYPE_CHECKING, List, Optional
 
 from anthropic import Anthropic
+from groq import Groq
 from PIL import Image
+from transformers import AutoTokenizer
```

Comment:
set Groq and AutoTokenizer within the class to make it optional

Reply:
What do you mean optional? In my understanding, the optional is usually used for the parameters?

Reply:
<img width="667" alt="Screenshot 2024-07-14 at 20 01 32" src="https://github.com/user-attachments/assets/542736cd-b60c-4140-9720-8e5d2dc0a146">
I mean move the import within the class like here, so that the library will only be imported when the class is used